### ISSUE TITLE
Philips hue colours change when you change brightness

### ISSUE THREAD

**Issue by matthijsvdr** [athombv/homey#2245]
**Metadata**

* **Homey Version**:
Latest stable
**If related to a Homey App:**
Philips Hue
* **Homey App Version**:
3.0.4

* **Affected Devices (preferably product codes)**:
Color bulbs

**What did you try to achieve?**
When changing the brightness of the hue lights the color also changes

1) In hue app set all lights to red
2) In homey set a scene (with differed colours)
2a) the colours do go to the correct scene and color
3) Let homey change the brightness of the lights (either over time or directly)
4) The lights will be set to the color of the first step

**What did you expect as result?**
Colours not to change back to the first step

**What was the result?**
Colours changed

**Any other remarks**

**Comment by WeeJeWel** [comment]
This is expected behavior, because Homey does not know what the color of the devices in the scene is. However, if you delay the brightness card with 10s, Homey did synchronise the colors back and it would work as expected.

**Comment by matthijsvdr** [comment]
@WeeJeWel 
> This is expected behavior

I don't agree with this, the expected behaviour would be that if I ask homey to set the brightness it sets the brightness and **not** the color.
Waiting for 10 seconds for homey to sync color (that does not work btw, 20 seconds does) is a workaround for the problem.

I mean Homey could just set only the brightness on the hue bridge:

```json
{
	"bri": 200
}

```

After all I'm not asking homey to set the color of the lights, but only the brightness.

**Comment by WeeJeWel** [comment]
I'll see if I can find a solution.
